mod-source-record-manager-server/src/main/resources/rules/rules.json

```diff
@@ -10,248 +10,12 @@
   "008": [
     {
       "target": "instanceTypeId",
-      "description": "Instance type based on LDR/06",
+      "description": "Default Unspecified Instance type in case when no 336 field present",
       "subfield": [],
       "rules": [
         {
-          "description": "Instance type for text",
-          "conditions": [
-            {
-              "type": "char_select",
-              "parameter": {
-                "from": 6,
-                "to": 7
-              },
-              "value": "a",
-              "LDR": true
-            }
-          ],
-          "value": "6312d172-f0cf-40f6-b27d-9fa8feaf332f"
```

Comment:
These ids are wrong, they were just overridden by rule 336 while mapping. Now we use id of the Unspecified type and override it by 336 rule